mssqlcli/mssql_cli.py

```diff
@@ -498,7 +524,7 @@ def _evaluate_command(self, text):
                 self.mssqlcliclient_main.execute_query(text):
 
             total = time() - start
-            if self._should_show_limit_prompt(status, rows):
+            if self._should_show_limit_prompt(status, rows) and self.interactive_mode:
                 click.secho('The result set has more than %s rows.'
```

Comment:
Can we move the self.interactive_mode check into the _should_show_limit_prompt method? This way, any caller in non-interactive mode will get false.